src/plugins/sync-title/client.js

```diff
@@ -33,15 +33,22 @@ function installClientMutation(sync) {
 
 }
 
-module.exports = function clientSyncTitle(client) {
-
-	function sync(value) {
-		client.sendEvent('title', value);
-	}
+module.exports = function(syncPage) {
+	function clientSyncTitle(client) {
+
+		function sync(value) {
+			var title = {
+				'syncPage' : syncPage,
+				'value' : value
+			};
```

Comment:
We have to be careful here, since there are going to be a bunch of old clients kicking around forever that are passing `title` as the second parameter. If we can do this in a backwards compatible way, that's best.

What'd I'd recommend is: `client.sendEvent('title', title, iframeOnly)`

Then the host can handle `iframeOnly` either being passed or being omitted. If it's missing (old client), then it defaults to `false` which is the old behaviour anyway.

Reply:
Yeah makes sense! I will update it.